charts/operator-wandb/charts/clickhouse/templates/_helpers.tpl

```diff
@@ -0,0 +1,285 @@
+                <access_management>1</access_management>
+                <profile>{{ include "wandb.clickhouse.user" $ }}</profile>
+            </default>
+        </users>
+        <user_directories>
+            <users_xml>
+                <path>/etc/clickhouse-server/users.xml</path>
+            </users_xml>
+        </user_directories>
+        <path>/var/lib/clickhouse/</path>
+        <tmp_path>/var/lib/clickhouse/tmp/</tmp_path>
+        <user_files_path>/var/lib/clickhouse/user_files/</user_files_path>
+        <format_schema_path>/var/lib/clickhouse/format_schemas/</format_schema_path>
+        <logger>
+            <level>debug</level>
+            <console>true</console>
+            <log remove="remove"/>
+            <errorlog remove="remove"/>
+        </logger>
+        <display_name>wandb_weave node_{{ $i }}</display_name>
+        <listen_host>0.0.0.0</listen_host>
+        <http_port>{{ include "wandb.clickhouse.port" $ }}</http_port>
+        <tcp_port>{{ $.Values.server.tcpPort }}</tcp_port>
+        <interserver_http_port>{{ $.Values.server.intrsrvhttpPort }}</interserver_http_port>
+        <distributed_ddl>
+            <path>/clickhouse/task_queue/ddl</path>
+        </distributed_ddl>
+        <remote_servers>
+            <wandb_weave>
+                <shard>
+                    <internal_replication>true</internal_replication>
+                    {{- include "clickhouse.serverNodes.replicas" $ | nindent 20 }}
+                </shard>
+            </wandb_weave>
+        </remote_servers>
+        <zookeeper>
+            {{- include "clickhouse.keeperNodes" $ | nindent 16 }}
+        </zookeeper>
+        <macros>
+            <shard>01</shard>
+            <replica>0{{ $i }}</replica>
+            <cluster>{{ $.Values.server.cluster }}</cluster>
+        </macros>
+        <merge_tree>
+            <storage_policy>s3_main</storage_policy>
+        </merge_tree>
+        <storage_configuration>
+            <disks>
+                <s3_disk>
+                    <type>s3</type>
+                    <endpoint>https://s3.{{ $.Values.buckets.region }}.amazonaws.com/{{ index $.Values.buckets.replicaBucketNames $i }}</endpoint>
+                    {{- if $.Values.buckets.accessKeyId }}
+                    <access_key_id>{{ $.Values.buckets.accessKeyId }}</access_key_id>
+                    <secret_access_key>{{ $.Values.buckets.secretAccessKey }}</secret_access_key>
+                    {{- else }}
+                    <use_environment_credentials>true</use_environment_credentials>
+                    {{- end }}
+                    <metadata_path>/var/lib/clickhouse/disks/s3_disk/</metadata_path>
+                </s3_disk>
+                <s3_cache>
+                    <type>cache</type>
+                    <disk>s3_disk</disk>
+                    <path>/var/lib/clickhouse/disks/s3_cache/</path>
+                    <max_size>20Gi</max_size>
```

Comment:
It's not a big deal for now, but if you could make this a configuration in the template, that would be great in case we need to tweak the cache size.
This value must be lower than ephemeral storage (persistence.size) by default.